src/Core/System/SystemConfig/Event/SystemConfigChangedEvent.php

```diff
@@ -15,8 +15,7 @@ class SystemConfigChangedEvent extends Event
      */
     public function __construct(
         private readonly string $key,
-        /** @deprecated tag:v6.7.0 - Will be natively typed */
-        private $value,
+        private mixed $value,
```

Comment:
```suggestion
        private array|bool|float|int|string|null $value,
```

same as in the other event